tests/metrics/test_metrics.py

```diff
@@ -139,3 +142,27 @@ def change_and_check_device_dtype(device, dtype):
         metric.half()
         assert metric.dtype == torch.float16
         assert metric(input1, input2).dtype == torch.float16
+
+
+@pytest.mark.skipif(torch.cuda.device_count() < 2, reason="test requires multi-GPU machine")
+@pytest.mark.parametrize("metric", [
+    DummyTensorMetric,
+    DummyNumpyMetric(),
+])
+def test_pickable(tmpdir, metric: Metric):
```

Comment:
can we also add a test, that simply tries to store and load the metric with `torch.save` and `torch.load` as this is a more direct test for this?

Reply:
added :)